ViewModel/Alternatives/Results/DamageWithUncertaintyVM.cs

```diff
@@ -127,7 +127,7 @@ private void LoadHistogramData(AlternativeComparisonReportResults altResults, in
 
             if (histogram != null)
             {
-                double[] binValues = histogram.BinCounts.Select(i => (double)i).ToArray();
+                double[] binValues = histogram.BinCounts.Select(i => (double)i/histogram.SampleSize).ToArray();
```

Comment:
It's not very obvious what's going on here.  The API should be doing this kind of logic, not you, why are we dividing it by the sample size?

Reply:
github issues made.